### PR TITLE
feat(#369, code-reviews): Add additional resource links

### LINKED ISSUE #369: [Feat]: Add links to resources section of code review standards
### Issue Description

The links provided within the '_resources_' section of the [code-reviews.md](https://github.com/Shift3/standards-and-practices/blob/main/standards/code-reviews.md) page are a little dated overall, and while still very useful, could use some additions. We have an excellent developer connect presentation that is extremely relevant, and it'd be nice to include that. 

### Proposed Solution

There should be two additional links provided in the resources section. 
1. Developer connect presentation from Corey Shuman
2. Link to article from Google on the importance of expedience during the code review process.

### Alternative Solutions

_No response_

### Additional context

This is my first issue written to the S&P board! Any and all feedback is welcome.

### PR DESCRIPTION
**Pull request by EthanKFoster**
## Changes
1. add link to BWTC dev connect on code review
2. add link to Google article emphasizing code review speed

## Purpose
#369 
This pr adds links to the resources section of the code review page. Applicable outside information was found that could be considered supplemental to our standards for code review, and find a good place here.  

## Approach
Additional links and resources bolster the strength of our standards, provided that the organization agrees with them.  

## Pre-Testing TODOs
NA

## Testing Steps
View the links and assess their value in including them in our S&P repo.

## Learning
Learning to navigate through the S&P and find places where I can contribute was the majority of this feature, and I'm looking forward to additional contributions I can make. 

Closes #369

### CODE REVIEW

File: standards/code-reviews.md [hunk 1]
```diff
@@ -88,3 +88,7 @@ Remember, this is a standard, so it isn't optional. _Never_ close your own Pull
 [How to become a better code reviewer](https://blog.asana.com/2016/12/7-ways-to-uplevel-your-code-review-skills/)
 
 [Digital Ocean's Effective Code Review Tips](https://blog.digitalocean.com/how-to-conduct-effective-code-reviews/)
+
+[Corey Shuman's Dev Connect on Code Review](https://playback.lifesize.com/#/publicvideo/9a9229d2-22b1-40db-9e8e-26308225128d?vcpubtoken=910e3bbe-cc9e-489d-b844-afe87baaccaa/)
+
+[Google 'Speed of Code Reviews' Article](https://google.github.io/eng-practices/review/reviewer/speed.html/)
```
Comment (amerikan) [inline standards/code-reviews.md:94]: the trailing slash might break the link

Comment (EthanKFoster) [reply standards/code-reviews.md:94]: Hey Erik! I'll remove the trailing slashes and recommit. Wasn't sure if that was a .md thing, as the other links in this document also have trailing slashes.

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
Always assign pr-team to your PRs, per our [code review](https://github.com/Shift3/standards-and-practices/blob/main/standards/code-reviews.md#process) process. Assign your PRs to[ @Shift3/pr-team](https://github.com/orgs/Shift3/teams/pr-team) in addition to the main devs on this project. This will automatically grab three developers from the PR team and assign them to perform a review. This will help to get a quicker turnaround on your reviews, and will also help this project to get reviews and feedback from team members with different perspectives and skill levels.

Example: https://github.com/Shift3/bitwiseindustries.com/pull/110

**Comment by EthanKFoster** [comment]
Hi @michaelachrisco!! 

I opted for forking when my fellow onboarding podmate Stephen Russell mentioned difficulty in branching and pushing directly to this repo. Additionally, when I requested another review from Erik on the issue above, it removed the other reviewers automatically, which I didn't expect, and I'm currently not able to add the PR team to the reviewer section (at least in the way I'm familiar from doing the same in our pod). 

I don't believe I have full collaborator access to this repo currently. I'm not able to add reviewers, assignees, labels, or any other features found in the sidebar.

**Comment by EthanKFoster** [comment]
@michaelachrisco Would the CircleCI plugin have been fine with forking if a branch had been made _prior_ to the fork?

**Comment by michaelachrisco** [comment]
Awesome! Looks like that was the issue. 

Ill make sure moving forward new devs are part of the PR-Team from day 1 (2?).

**Comment by EthanKFoster** [comment]
You rock, Mike! I appreciate your quick responses, definitely ramped up my own learning process in the beginning here. Thanks for the help 👍🏻

File: standards/code-reviews.md [hunk 2]
```diff
@@ -88,3 +88,7 @@ Remember, this is a standard, so it isn't optional. _Never_ close your own Pull
 [How to become a better code reviewer](https://blog.asana.com/2016/12/7-ways-to-uplevel-your-code-review-skills/)
 
 [Digital Ocean's Effective Code Review Tips](https://blog.digitalocean.com/how-to-conduct-effective-code-reviews/)
+
+[Corey Shuman's Dev Connect on Code Review](https://playback.lifesize.com/#/publicvideo/9a9229d2-22b1-40db-9e8e-26308225128d?vcpubtoken=910e3bbe-cc9e-489d-b844-afe87baaccaa)
+
+[Google 'Speed of Code Reviews' Article](https://google.github.io/eng-practices/review/reviewer/speed.html)
```
Comment (michaelachrisco) [inline standards/code-reviews.md:94]: What do you think of moving this document/article into our own S&P. Some of these tips could be useful for the rest of the team, especially the "fast PRs" and benifits thereof.

**Review by michaelachrisco — APPROVED** [review]
Awesome, I really like us adding more video resources to our S&P!

**Pull request merged**